lib/eclint.ts

```diff
@@ -93,6 +93,10 @@ module eclint {
 		infer(line: linez.Line): any;
 	}
 
+	export interface Line extends linez.Line {
```

Comment:
Do you think it makes sense for this feature to be baked into the [`linez`](https://github.com/jedmao/linez) package? I'm the project owner of linez too, so it should be painless.

If so, I think it best that it be enabled via a configuration option, like so:

```js
linez(contents, {
    docCommentStart: '/**',
    docCommentEnd: '*/'
});
```
What do you think?

We can still use a regex internally with `new RegExp('^\\s*' + docCommentStart)`.

Anyway, I think I'd rather do that then have some special `eclint.Line`.